model/project_ref.go

```diff
@@ -257,7 +257,7 @@ func FindOneProjectRef(identifier string) (*ProjectRef, error) {
 		projectRef,
 	)
 	if adb.ResultsNotFound(err) {
-		return nil, nil
+		return nil, errors.New("error finding the project")
```

Comment:
same comment about including the project ID

Reply:
This one is used by context.go and is not displayed to the user. 
context.go checks if the errors.Cause(err) is equal to "error finding the project", so adding the projectID would make it harder to do so.

Reply:
Ah, that makes sense. Shouldn't it use the constant, then?

Reply:
It should. I'll fix it. Thank you!